app/src/App.tsx

```
@@ -0,0 +1,51 @@
+
+  async function greet() {
+    // Learn more about Tauri commands at https://tauri.app/develop/calling-rust/
+    setGreetMsg(await invoke("greet", { name: name() }));
+  }
+
+  return (
+    <main class="container">
+      <h1>Welcome to Tauri + Solid</h1>
+
+      <div class="row">
+        <a href="https://vitejs.dev" target="_blank">
```

Biome (1.9.4)

[error] Avoid using target="_blank" without rel="noreferrer".

Opening external links in new tabs without rel="noreferrer" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noreferrer" attribute.


(lint/a11y/noBlankTarget)